extensions/wikia/GlobalNavigation/scripts/GlobalNavigationTracking.js

```diff
@@ -6,39 +6,76 @@ $(function($) {
 	$globalNavigationSearch = $globalNavigation.find('#searchForm');
 	searchSuggestionsShowed = false;
 
+	/**
+	 * Tracking helper function with most commonly used options. Overrides are passed in by callers.
+	 */
+	track = Wikia.Tracker.buildTrackingFunction({
+		category: 'search',
+		trackingMethod: 'both',
+		action: Wikia.Tracker.ACTIONS.CLICK
+	});
+
+	/**
+	 * Parent click handler for events in the global nav
+	 * @param {object} event
+	 */
 	function globalNavigationClickTrackingHandler(event) {
-		var $element, label, $parent;
+		var $element, label;
 
 		//Track only primary mouse button click
 		if (event.type === 'mousedown' && event.which !== 1) {
 			return;
 		}
 
-		track = Wikia.Tracker.buildTrackingFunction({
-			action: Wikia.Tracker.ACTIONS.CLICK,
-			category: 'top-nav',
-			trackingMethod: 'ga'
-		});
-
 		$element = $(event.target);
-		if ($element.closest('.wikia-logo').length > 0) {
-			label = 'wikia-logo';
-		} else if ($element.closest('.hub-links').length > 0 || $element.closest('.hub-list').length > 0) {
+
+		label = getLabelByIdentifier($element) ||
+			getLabelByDOM($element);
+
+		if (label !== false) {
+			track({
```

Comment:
Did you mean to send the tracking for top-nav to both here? The last version was only sending it to ga.